lib/api/controllers/routerController.js

```diff
@@ -54,7 +54,7 @@ module.exports = function RouterController (kuzzle) {
       executeFromRest.call(kuzzle, params, request, response);
     });
 
-    api.get('/_getStats', function (request, response) {
+    api.post('/_getStats', function (request, response) {
```

Comment:
ihmo, GET is the better way to retrieve content when there is just one parameter, it's more HTTP-logic compliant (GET for fetching content, POST for sending content or executing an action).
But... for more complex queries (like for an advanced search, for example), it's not possible to do it properly with GET so we have to do a POST anyway..

So, for the getStats query, it would be OK for me to do a POST if we can have more than one filter parameters.... for example, it could be interesting to have, at least:
* startTime (required, or just act like a getLastStats if not given).
* stopTime (optional, use "now" by default).

What do you think about this suggestion ?